packages/components/src/components/Select/SelectMenu.tsx

```diff
@@ -37,9 +37,30 @@ export type SelectMenuProps = {
      * The filter text shown in the combobox input element.
      */
     filterText?: string;
+    /**
+     * @internal
+     * The type of the menu.
+     */
+    selectedItem?: {
+        value: string;
+    } | null;
```

Comment:
why both `| null` and optional?

Reply:
For whatever reason `useCombobox` uses null and `useSelect` uses undefined (downshift hooks)